apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -77,6 +78,7 @@ public static Builder builder() {
   private final CacheControl defaultCacheControl;
   private final CacheHeaders defaultCacheHeaders;
   private final ApolloLogger logger;
+  private final ApolloCallTracker callTracker = new ApolloCallTracker();
```

Comment:
What I meant is if user doesn't care about keep tracking of calls, then why should we create it at all and have some synchronization overhead.

But at the same time for bigger picture we probably need it.
Then I would call it not a tracker but `ActiveCallRegistry` or smth like this, and  it can be used not only for tracking how many calls we are running right now but potentially for batching requests. In this case we need to include as well watchers.